### ISSUE TITLE
Vertical Menu Issue

### ISSUE THREAD

**Issue by xcrap** [Semantic-Org/Semantic-UI#5454]
I'm having a issue I can't seem to solve with a vertical menu, I'm using the simple dropdown without javascript but this is how it displays:

![screen shot 2017-06-09 at 02 58 02](https://user-images.githubusercontent.com/828596/26959401-8f8d762c-4cbf-11e7-8b58-bbfeed57a835.png)

The submenus goes one line down.
This is my code.

```
<div class="ui fluid blue vertical menu">
    <a class="item" href="">Stufff</a>  
    <a class="item" href="">Stufff</a>
    <div class="ui simple dropdown item">Submenu
        <i class="dropdown icon"></i>
        <div class="menu">
            <a class="item" href="">Stufff</a>
    </div>
</div>
```

Removing the simple and using the Javascript API it works fine.
The problem comes as soon as you put the simple in the vertical menu variation.
In the documentation near the https://semantic-ui.com/collections/menu.html#vertical-menu there's this.

> Dropdown Item
> An item may contain a nested menu in a dropdown.
> To have a dropdown open without Javascript, use the simple variation

But in the code presented there's no simple class, it uses the javascript approach.
It's my mistake or there's no vertical submenu with no javascript solution ?

**Comment by Banandrew** [comment]
Hi @xcrap, I think you’re right that it doesn’t exist. I’ll open a PR as soon as I can, but it will take some time to be merged, so you might want to add the missing rules immediately in `installation_folder/src/site/modules/dropdown.overrides`; don’t forget to rebuild the framework afterwards.

**Comment by Banandrew** [comment]
@xcrap I was wrong, the rules actually exist. The details are in the PR; you need to override the rule in `installation_folder/src/site/collections/menu.overrides` instead.

**Comment by xcrap** [comment]
Perfect. Since I'm using the compiled CSS version I'll override the styles outside in external CSS I have for this specific project.

**Comment by lubber-de** [comment]
Fixed in upcoming 2.7.2 https://github.com/fomantic/Fomantic-UI/pull/339
